front/src/Phaser/Game/GameMap.ts

```diff
@@ -8,15 +8,40 @@ export type PropertyChangeCallback = (newValue: string | number | boolean | unde
  * It is used to handle layer properties.
  */
 export class GameMap {
-    private key: number|undefined;
-    private lastProperties = new Map<string, string|boolean|number>();
+    private key: number | undefined;
+    private lastProperties = new Map<string, string | boolean | number>();
     private callbacks = new Map<string, Array<PropertyChangeCallback>>();
+
+    /**
+     * tileset.firstgid => (Map<tileid,Array<ITiledMapLayerProperty>>)
+     */
+    private tileSetPropertyMap = new Map<number, Map<number, Array<ITiledMapLayerProperty>>>()
     public readonly layersIterator: LayersIterator;
 
+    public exitUrls: Array<string> = []
+
     public constructor(private map: ITiledMap) {
         this.layersIterator = new LayersIterator(map);
+
+        for (const tileset of map.tilesets) {
+            if (!this.tileSetPropertyMap.has(tileset.firstgid)) {
+                this.tileSetPropertyMap.set(tileset.firstgid, new Map())
+            }
+            tileset?.tiles?.forEach(tile => {
+                if (tile.properties) {
+                    this.tileSetPropertyMap.get(tileset.firstgid)?.set(tile.id, tile.properties)
+                    tile.properties.forEach(prop => {
+                        if (prop.name == "exitUrl" && typeof prop.value == "string") {
```

Comment:
i thought since the other one is deprecated anyways we dont need to add the deprecated functionality to this mode

Reply:
:+1: